keps/sig-cli/4292-kubectl-debug-custom-profile/kep.yaml

```diff
@@ -18,25 +18,22 @@ see-also:
   - "/keps/sig-cli/1441-kubectl-debug"
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: alpha
+stage: stable
 
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.31"
+latest-milestone: "v1.32"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
   alpha: "v1.30"
   beta: "v1.31"
-  stable: "TBD"
+  stable: "1.32"
 
 # The following PRR answers are required at alpha release
 # List the feature gate name and the components for which it must be enabled
-feature-gates:
```

Comment:
Don't remove it. Even in code we'll need to lock it to true, similarly how we handle regular feature gates.

Reply:
Added back.